Sources/swift-parser-cli/Commands/PerformanceTest.swift

```diff
@@ -37,19 +41,39 @@ struct PerformanceTest: ParsableCommand {
       .filter { $0.pathExtension == "swift" }
       .map { try Data(contentsOf: $0) }
 
-    let start = Date()
+    var incrementalParseTransition: IncrementalParseTransition? = nil
+    var incrementalParseAffectRangeCollector: IncrementalParseNodeAffectRangeCollector? = nil
+
+    var totalTime: TimeInterval = .zero
     let startInstructions = getInstructionsExecuted()
-    for _ in 0..<self.iterations {
+    var previousTreeDict: [Data: (SourceFileSyntax, IncrementalParseNodeAffectRangeCollector?)] = [:]
```

Comment:
I think we should still store a `(SourceFileSyntax, IncrementalParseNodeAffectRangeCollector)` here since we could not get the collector from `IncrementalParseTransition`.

Reply:
But we could just pass `nil` for the `parseNodeAffectRange` because we aren’t doing another incremental parse based on the tree produced in this loop, right?

Reply:
Yeah, but we still need to store a `IncrementalParseNodeAffectRangeCollector ` for every file if we are doing incremental parse, right? I store a `(IncrementalParseTransition, IncrementalParseNodeAffectRangeCollector)` here in the new commit and it should make the actual performance test loop simpler.

Reply:
Why do we need to store an `IncrementalParseNodeAffetRangeCollector`? Can’t you just change the call in the performance measurement loop to pass `nil` for `parseNodeAffectRange` because we aren’t doing another incremental parse based on the tree that’s generated in the performance measurement loop?

Reply:
But if we pass `--incremental-parse` to performance test, we would need to use the `IncrementalParseNodeAffetRangeCollector` set up in the initial parse to enable incremental parse in the performance measurement loop. Do you mean we would not measure the performance of incremental parse when `self.iterations` > 1?